src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.tsx

```diff
@@ -5,22 +5,32 @@ import {omit} from 'lodash';
 import {css} from 'react-emotion';
 
 import SentryTypes from 'app/sentryTypes';
-import AsyncComponent from 'app/components/asyncComponent';
+import AsyncComponent, {AsyncComponentState} from 'app/components/asyncComponent';
```

Comment:
I think what would be better here is to do `AsyncComponent['state']` using the [lookup type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#keyof-and-lookup-types) Instead of importing the type

Reply:
makes sense